# bounding box

#the pose of the box frame
geometry_msgs/PoseStamped pose_stamped

#the dimensions of the box
geometry_msgs/Vector3 dimensions
